docs/bridges.md

```diff
@@ -203,23 +203,24 @@ The pipeline may be triggered by invoking the Make target
 `refresh-bridges` (whilst in the in `src/ontology` directory, where the
 Makefile is located). Here is what happens then:
 
-1. Production of a SSSOM mapping set from the Uberon and CL
-   cross-references. The Uberon `-edit` file is merged with the mirrored
-   copy of CL (which is downloaded if not already available) and
-   cross-references from both Uberon and CL are extracted and turned
-   into a SSSOM mapping set. This takes care of all the mappings with
-   the foreign ontologies for which Uberon and CL are the source of
-   truth.
+1. Production of a SSSOM mapping set from the Uberon cross-references.
+   Cross-references are extracted directly from the Uberon `-edit` file
```

Comment:
I am wondering if long term, the choice of using the edit file as the basis for extracting mapping sets intended for release will bite us in the tail. There are a small handful of pipelines I know of which process edit file mappings into something else during the release process, and it is that "something else" that you would want to see in the published mapping set.

Reply:
If/when needed, it would be possible to extract the cross-references from the _preprocessed_ file (`EDIT_PREPROCESSED`) instead of the `-edit` file. That would still be compatible with the scenario where the mapping sets are needed to produce a component needed to produce the final release artefacts.

And where the mapping sets are _not_ needed to produce a component needed to produce the final release artefacts, there shouldn’t be any problem in extracting the cross-references from the final release artefacts.

Reply:
The only situation where we could paint ourselves in a corner is if we have a mapping set that, for any reason, must be extracted near the end of the pipeline (e.g. because it depends on some custom processes that happen late in the pipeline) _and_ that is also needed to derive a component that must be available early in the pipeline. That would obviously be a problem.